### PR TITLE
feat(drivers): Bootstraps CrateDB driver

### PR DESCRIPTION
**Pull request by cristipp**
**Check List**
- [x] Tests has been run in packages where changes made if available
- [x] Linter has been run for changed code
- [x] Tests for the changes have been added if not covered yet
- [x] Docs have been added / updated if required

**Issue Reference this PR resolves**

N/A

**Description of Changes Made (if issue reference is not provided)**

Bootstraps CrateDB driver.
See https://crate.io
See #4645

### CODE REVIEW

File: packages/cubejs-testing-shared/src/db/crate.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { GenericContainer } from 'testcontainers';
+
+import { DbRunnerAbstract, DBRunnerContainerOptions } from './db-runner.abstract';
+
+export class CrateRunner extends DbRunnerAbstract {
+  public static startContainer(options: DBRunnerContainerOptions) {
+    const version = process.env.TEST_QUEST_DB_VERSION || '5.0.0';
```
Comment (proddata) [inline packages/cubejs-testing-shared/src/db/crate.ts:7]:
```suggestion
    const version = process.env.TEST_CRATE_DB_VERSION || '5.0.0';
```

**Comment by proddata** [comment]
Hi @cristipp 

Georg from Crate.io here. 
I was pointed towards this PR. Cool that you are adding support :) 

Can I be of any help?

File: packages/cubejs-testing-shared/src/db/crate.ts (hunk 1 above)
Comment (cristipp) [reply packages/cubejs-testing-shared/src/db/crate.ts:7]: Done!

**Comment by cristipp** [comment]
Hi @proddata. Most of the work was done by @robd003, see https://github.com/robd003/cube.js/tree/add_crate. I'm just cleaning up some minor lint issues and adding a couple tests.

**Comment by cristipp** [comment]
Thanks for taking a look, @buntarb.
@proddata, feel free to DM me and/or create subsequent PRs. Hopefully they will be focused and smaller than the boostrap boilerplate

**Pull request merged**